db.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+
+// A singleton to ensure we only start the database once
+// assign the MongoMemoryServer instance to mongoServer
+let mongoServer;
+const uri = 'mongodb+srv://Surya:sun@asapcluster.3osvgms.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Avoid hardcoding the MongoDB URI in the code. It's a good practice to use environment variables for sensitive data. This can be achieved by using `process.env` in Node.js. [important]